cluster/config-defaults.yaml

```diff
@@ -18,8 +18,9 @@ cluster_autoscaler_release: "1_18"
 cluster_autoscaler_keep_backed_off_pools_scaled_up: "true"
 
 # How long to wait for pod eviction when scaling down.
-{{if eq .Environment "production"}}
+{{if eq .Cluster.Environment "production"}}
 cluster_autoscaler_max_pod_eviction_time: "1h"
+cluster_autoscaler_disable_node_instances_cache: "false"
```

Comment:
This was removed in another PR, not sure why it's added back.

Reply:
I will rebase on dev again

Reply:
removed again, whyever this was not fixed by the rebase